src/ILLink.Shared/TrimAnalysis/HandleCallAction.cs

```diff
@@ -254,7 +254,7 @@ GenericParameterValue genericParam
 			case var callType when (callType == IntrinsicId.Type_GetConstructors || callType == IntrinsicId.Type_GetMethods || callType == IntrinsicId.Type_GetFields ||
 				callType == IntrinsicId.Type_GetProperties || callType == IntrinsicId.Type_GetEvents || callType == IntrinsicId.Type_GetNestedTypes || callType == IntrinsicId.Type_GetMembers)
 				&& calledMethod.IsDeclaredOnType ("System.Type")
-				&& calledMethod.HasParameterOfType (0, "System.Reflection.BindingFlags")
+				&& calledMethod.HasParameterOfType ((NonThisParameterIndex) 0, "System.Reflection.BindingFlags")
```

Comment:
Not necesarilly in this PR - but eventually we should switch this to ILParameterIndex as well.

Reply:
Agreed, there's still a lot more work to do, I just thought it'd be best to separate it into multiple PR's.